main.tex

```diff
@@ -18,6 +18,21 @@
 \begin{document}
 
 \maketitle
+\subsubsection Internetverbreitung
+
+In Indien gibt es 2015 schon 350 millionen Menschen\autocite{slideshareIndia} , welche das Internet aktiv nutzen, und davon sind 159 millionen per Handy im Internet unterwegs. Somit sind sie schon auf Platz 2 der aktiven Internetnutzer weltweit. So groß sich die Zahl auch anhören mag, das sind lediglich 27\% der Bevölkerung\autocite{InternetCountry}. 
+
+Laut einer Studie sei 2015 ein PC in Indien ein Luxusgut, welches durch die schnelle Entwicklung des Handys für die Jugend schnell die Bedeutung verlor, und Handys mit Internet in der Gesellschaft allgegenwärtig wurde. Durch die explosionsartige Verbreitung der neuen Möglichkeit mit anderen zu kommunizieren und sich auszutauschen bauten sich in kurzer Zeit große Gruppen im Internet auf, um sich über neue Technologien, Wissenswertes und Interessen auszutauschen. Dieser rapide Aufbau von Gesellschaften ist ein neues Phänomen, welches vorallem in "Entwicklungsländern" auftritt, in denen neue und vergleichsweise preiswerte Technologien den Zugang zum Internet ermöglichen.   \textcite{empowermentThroughFacebook}
+
+\medskip
+Die durchschnittliche Bandbreite des mobilen Netzes war mit 2.8 Mb/s der durchschnittlichen festen Internetleitung überlegen, denn diese betrug lediglich 2.3 Mb/s\autocite{slideshareIndia}.
```

Comment:
```suggestion
Die durchschnittliche Bandbreite mobilen Internets war mit 2.8 Mb/s der durchschnittlichen Bandbreite eines Festanschlusses, von 2.3 Mb/s, überlegen \autocite{slideshareIndia}.
```

Reply:
Gutes Beispiel bei dem die Fakten, anstatt deren Auswirkungen im Mittelpunkt stehen.
Diese puren Fakten könnten gut in einen Text eingepflegt werden, z.B.:

Ein Aspekt, weshalb die mobilen Endgeräte auch allgegenwärtig wurden, ist deren Verbindungsgeschwindigkeit, welche mit 2.8 Mb/s der eines Festanschlusses (2.3) überlegen ist.

---

Auf dem Land wird  WLAN übrigens auch über das mobil Netz betrieben, das es dort i.d.R. keine Kabelanschlüsse in die Häuser gibt.